test/plugin.test.ts

```diff
@@ -103,6 +103,7 @@ describe("Github API best practices", function () {
         write: new Bottleneck.Group({ minTime: 50 }),
         onAbuseLimit: () => 1,
         onRateLimit: () => 1,
+        connection: new Bottleneck({ minTime: 50 }),
```

Comment:
Adding a test with `connection` property to add coverage.

I tried to add a proper test for this but I'm not sure how to assert the `expeiration` property is added to the limiter. Looks a bit complex to mock to me.

Reply:
I think this is good enough 👍🏼